### PR TITLE
Add more BMW software version definitions

### PR DESCRIPTION
**Pull request by polybassa**
Just a few more definitions

### CODE REVIEW

File: scapy/contrib/automotive/bmw/definitions.py [hunk 1]
```diff
@@ -252,10 +251,60 @@ def i2repr(self, pkt, x):
...
+        "0xC0": "SWUP",
+        "0xC1": "SWIP",
+        "0xA0": "ENTD",
+        "0xA1": "NAVD",
+        "0xA2": "FCFN",
+        "0x04": "GWTB",
+        "0x0D": "SWFK",
+    }
+    """
+        "HWEL", "Hardware (Elektronik)"
```
Comment (guedou) [inline scapy/contrib/automotive/bmw/definitions.py:281]: Will it make sense to add the English translation as a comment?

**Review by guedou — COMMENTED** [review]
Thanks for the PR.

File: scapy/contrib/automotive/bmw/definitions.py (hunk 1 above)
Comment (polybassa) [reply scapy/contrib/automotive/bmw/definitions.py]: done

**Review by guedou — APPROVED** [review]
Thanks!

**Pull request merged**